widgets/src/common/SVGWidget.js

```diff
@@ -124,7 +133,9 @@
 
     SVGWidget.prototype.update = function (domeNode, element, d) {
         Widget.prototype.update.apply(this, arguments);
-        element.attr("transform", "translate(" + this._pos.x + " " + this._pos.y + ")");
+        //element.attr("transform", "translate(" + this._pos.x + " " + this._pos.y + ")");
+        element.attr("transform", "translate(" + this._pos.x + " " + this._pos.y + ")scale(" + this._scale + ")");
```

Comment:
Interestingly the "transform" attribute will never have any affect as it gets written in the base class already.